api/http/mypy.ini

```diff
@@ -0,0 +1,11 @@
+[mypy]
+python_version = 3.9
+strict = True
```

Comment:
Looks a little bit strange — strict mode is on, but I don't see `# typing: ignore` directives, which should be required at least for Indigo API, since it's untyped at the moment. Am I missing something here?

Reply:
I'm just turning off all indigo imports using `ignore_missing_imports = True` 
https://github.com/epam/Indigo/pull/502/files/0e7dc9f30fe65ef0e93326eb92d6791acdc2549c#diff-fe5c8c88597291fc7f9219d3411417e39b5da69faf2d8b748cf2c6b882cab8f0R7